### PR TITLE
Firebase setup and first redirections

### PR DESCRIPTION
**Pull request by camh-**
Add the initial firebase config generated with `firebase init` and tweaked.

Add a jsonnet config file that is used to generated the `json` config file
with redirections configured for `jsonnext` (module and files) and
`s` (module only).

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      }
+    </style>
+  </head>
+  <body>
+    <div id="message">
+      <h2>404</h2>
+      <h1>Page Not Found</h1>
+      <p>The specified file was not found on this website. Please check the URL for mistakes and try again.</p>
+      <h3>Why am I seeing this?</h3>
+      <p>This page was generated by the Firebase Command-Line Interface. To modify it, edit the <code>404.html</code> file in your project's configured <code>public</code> directory.</p>
```
Comment (juliaogris) [inline public/404.html]: I'd at least delete the above two lines... I also think we need a tiger on the 404 page ;)

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Foxygoat</title>
+  </head>
+  <body>
+    <img src="stay-healthy-tiger.jpeg">
```
Comment (juliaogris) [inline public/index.html:7]: I see... already there.

File: firebase.libsonnet [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    // For example, you can import a jsonnet file using
+    // `foxygo.at/jsonnext/importer` using
+    // `//foxygo.at/jsonnext/master/jlib/utils.libsonnet` as a filename. This
+    // gets redirected to a raw github file in the repo under the `master` ref.
+    files(repo):: self.temporary("%s/:path*" % repo, self.files_template % repo),
+};
+
+local html = {
+    index(module_name): |||
+        <html><head>
```
Comment (juliaogris) [inline firebase.libsonnet]: Could you add an html comment: generated by jsonnet or something, please?

File: public/404.html (hunk 1 above)
Comment (camh-) [reply public/404.html]: ok, can do. can Mali do a sad-not-found-tiger?

File: public/index.html (hunk 2 above)
Comment (camh-) [reply public/index.html:7]: https://foxygo.at/

File: firebase.libsonnet (hunk 3 above)
Comment (camh-) [reply firebase.libsonnet]: done

**Comment by camh-** [comment]
@juliaogris PTAL

**Review by juliaogris — APPROVED** [review]

**Comment by camh-** [comment]
I've push some more changes, but they are trivial and I wont ask for re-review. I added the `fmt` target to format the jsonnet files and ran it. So there are inconsequential changes to the jsonnet (whitespace and quoting) and just one rule in the Makefile.

**Pull request merged**